src/core/utils.js

```diff
@@ -360,6 +362,27 @@ export function isCompatibleWithUserAgent({
         https://github.com/mozilla/addons-frontend/issues/2074`);
     }
 
+    // Check to see if:
```

Comment:
Wait, you're marking the min version here as 57 but not the max version, which is what this issue is for. This issue fixes extensions that aren't 57-compatible being marked as such because we *don't* and *shouldn't* actually honour the `maxVersion` attribute.